src/services/scenarioRun/ScenarioRunService.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import fileDownload from 'js-file-download';
+import { LOG_TYPES } from "./ScenarioRunConstants";
+
+class ScenarioRunService {
+  constructor (apiService, organizationId) {
+    this.apiService = apiService;
+    this.organizationId = organizationId;
+    this.scenarioRunApi = new apiService.ScenariorunApi();
+  }
+
+  async downloadCumulatedLogsFile (lastRun) {
+    try {
+      const logs = await this.scenarioRunApi.getScenarioRunCumulatedLogs(this.organizationId, lastRun.scenarioRunId);
+      const blob = new Blob([logs]);
+      fileDownload(blob, lastRun.scenarioRunId + '_cumulated_logs.txt');
+    } catch (e) {
+      console.error(e);
+    }
+  }
+
+  async downloadLogsSimpleFile (lastRun) {
```

Comment:
Since this branch comes with breaking changes, could we take the opportunity to rename some functions ?
`downloadLogsSimpleFile` could be renamed to `downloadSimpleLogsFile` for consistency with `downloadLogsFile` and `downloadCumulatedLogsFile`